src/extensionConfiguration.ts

```diff
@@ -0,0 +1,16 @@
+import { loadConfigurationForExtension } from '@codingame/monaco-editor-wrapper/dist/features/extensionConfigurations'
+import staticOptions, { LanguageClientId } from './staticOptions'
+
+export async function loadExtensionConfigurations (forLanguageClientIds: LanguageClientId[], nonMutualizableOnly: boolean): Promise<void> {
```

Comment:
Since you only check "not nonMutualizableOnly", you should change the variable to `mutualizableOnly`.

Reply:
I hate double negations :)

Reply:
is it better? b2861e2d7d80f6a1b9ed422c2de75c3ec54debce

Reply:
+ 4e319dd8a58b6f66e71592a96d00f9265ebe0d16